src/lang/Messages.properties

```diff
@@ -918,6 +918,12 @@ core.api.action.sendRequest = Sends the HTTP request, optionally following redir
 core.api.action.setMode = Sets the mode, which may be one of [safe, protect, standard, attack]
 core.api.action.setOptionDnsTtlSuccessfulQueries = Sets the TTL (in seconds) of successful DNS queries (applies after ZAP restart).
 core.api.action.shutdown = Shuts down ZAP
+core.api.action.addProxyChainExcludedDomain = Adds a domain to be excluded from the outgoing proxy, using the specified value and (optionally) if enabled (default, true) and if it's a regex (default, false).
+core.api.action.modifyProxyChainExcludedDomain = Modifies a domain excluded from the outgoing proxy. Allows to modify the value, if enabled or if a regex. The domain is selected with its index.
+core.api.action.removeProxyChainExcludedDomain = Removes a domain excluded from the outgoing proxy, with the given index.
```

Comment:
For this one and the one before it, should we have some kind of `See also core.api.view.proxyChainExcludedDomains`? So that users know how to get the index?